### DISCUSSION TITLE
Chart with multiple button to switch date

### DISCUSSION THREAD

**Discussion by rcheiko** [apertureless/vue-chartjs#997] [Q&A]
Hello,
I want to create a chart where you can switch the time of the chart
example :
1 day / 1 week / 1 month / all
There is something that is implemented in vue chartjs or i just need to re create a chart for every date ?
Thank you !

**Comment by apertureless** [comment]
You need to either re-create the chart with the new data or create 3 charts and conditionally hide or show them
